Tasks/SqlAzureDacpacDeployment/Utility.ps1

```diff
@@ -9,6 +9,29 @@ function Check-ServerName
     }
 }
 
+function Get-FormattedSqlUsername
```

Comment:
As discussed we can take deployment group task related changes later.
Added L0 for "@" symbol. We are not treating any other characters in a special way.